### ISSUE TITLE
Pretty neat, but it lacks styles management

### ISSUE THREAD

**Issue by jkomyno** [lubien/koa-react-boilerplate#13]
I really appreciated the protected route and the authentication handling, however I believe that adding some sort CSS module to manage local styles (which could also be compiled from SASS/LESS/Stylus with a grunt/gulp/webpack script) would be a huge help and a valid improvement.

**Comment by lubien** [comment]
Indeed it is. This week I'm really in a hurry. Could you do this? Or if you can wait a little bit.
That could be a good PR for the [Hacktoberfest](https://hacktoberfest.digitalocean.com).

**Comment by jkomyno** [comment]
Indeed. Git clone'd, let's see if I can handle this, it should be a nice exercise to eventually strengthen my webpack abilities

**Comment by lubien** [comment]
Any issues let me know. I may help as I can.

**Comment by REPTILEHAUS** [comment]
Are you guys just looking for a Gulp or Gruntfile that compiles SASS to CSS ?

**Comment by lubien** [comment]
Wouldn't be best to handle styles with webpack since it's the main build tool there?

**Comment by REPTILEHAUS** [comment]
Just thought I would check as those build scripts were mentioned, no worries

**Comment by lubien** [comment]
No problem ^^
